authentication-oauth2.0.md

```diff
@@ -68,4 +77,192 @@ Example
 
 ```bash
 curl -H "Authorization: Bearer ACCESS-TOKEN" https://app.solvexia.com/api/v1/processes
+```
+
+## Authorise Code Flow
+
+Authorise Code Flow must be used in wll third-party applications that intended to go public.
```

Comment:
Authorise Code Flow must be used in **all** third-party applications that intended to go public.

Reply:
Thanks @bhavin07 this is fixed now.

Reply:
Thanks @bhavin07. Fixed.